src/pages/about.tsx

```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+import $typo from "@/styles/Typography.module.css";
+
+const About = () => {
+  return (
+    <>
+      <h2>About Compass</h2>
+
+      <div>
+        Compass is a civic app we are building for the San Francisco Unified
+        School District. The app seeks to assist Teachers with setting and
+        tracking goals for students with disabilities.
+      </div>
+      <div></div>
+    </>
```

Comment:
Is there a reason we have the fragments instead of divs? (It makes it easier to apply CSS if they are specific elements rather than fragments.)
Minor cleanup request - please remove extra code we are not currently using, like the imports and empty div - we can add needed imports and components later.
Good job - this is a good starting point.